common/djangoapps/student/models.py

```diff
@@ -449,6 +450,9 @@ class Meta(object):
     name = models.CharField(blank=True, max_length=255, db_index=True)
 
     employee_id = models.CharField(blank=True, max_length=12, db_index=True)
+    organization = models.ForeignKey(
+        Organization, db_index=True, related_name='user_profiles', on_delete=models.CASCADE, null=True, blank=True
```

Comment:
can a one user have multiple orgs?

Reply:
no, one user can be linked to one organization but one organization can have multiple users, which results in 1-Many relation. That's why using ForeignKey instead of OneToOne.